x-pack/plugins/lens/public/pie_visualization/toolbar.tsx

```diff
@@ -187,7 +187,7 @@ export function PieToolbar(props: VisualizationToolbarProps<PieVisualizationStat
           <EuiHorizontalRule margin="s" />
           <EuiFormRow
             label={i18n.translate('xpack.lens.pieChart.percentDecimalsLabel', {
-              defaultMessage: 'Decimal places for percent',
+              defaultMessage: 'Maximum decimal places for percent',
             })}
             fullWidth
             display="columnCompressed"
```

Comment:
The label is really long and just isn't fitting well in this space as the column display. Let's just change this to  
```suggestion
            display="rowCompressed"
```